spec/acceptance/nodesets/ubuntu-server-1404.yml

```diff
@@ -1,7 +1,7 @@
 HOSTS:
-  ubuntu-server-1404-x64:
```

Comment:
Why do we have Ubuntu and Ubuntu Server? I don't get the distinction.

Reply:
I think the `server` was left overs from previous vagrant boxes.  I don't know of a difference, the containers are all just `ubuntu`, no server distinction.

Reply:
Please remove all the cruft we don't need.